<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>credit</class>
 <widget class="QDialog" name="credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>430</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Ежемесячный платеж:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Differentiated">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>520</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Differentiated</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_credit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>490</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 1px solid white;
   border-radius: 20px;
   font: 20pt &quot;Cochin&quot;;
   background-color: rgb(81, 96, 76);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #112617, stop: 1 #112617);
    color: white;
    border: 2px solid rgb(81, 96, 76);
}</string>
   </property>
   <property name="text">
    <string>Count</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Срок (месяц):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Переплата:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_interes_rate">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>160</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 30pt &quot;Cochin&quot;;
background-color: rgb(81, 96, 76);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_monthly_payment">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>430</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 30pt &quot;Cochin&quot;;
background-color: rgb(81, 96, 76);</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Annuity">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>490</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuity</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>280</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Input</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Общая сумма оплаты:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_total_credit_amount">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>60</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 30pt &quot;Cochin&quot;;
background-color: rgb(81, 96, 76);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_term">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>110</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 30pt &quot;Cochin&quot;;
background-color: rgb(81, 96, 76);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>490</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Тип:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Процентная ставка:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_total_payment">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>330</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 30pt &quot;Cochin&quot;;
background-color: rgb(81, 96, 76);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_overpayment">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>380</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 30pt &quot;Cochin&quot;;
background-color: rgb(81, 96, 76);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Общая сумма кредита:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
